omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
```

Comment:
This will be updated to have the support status for 5.0/5.1/5.2 in the next push.